testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -213,6 +214,47 @@ def test_group_return_unsorted_sorted_unique(self, ugroup):
         assert unsorted_unique is sorted_unique
         assert unique._cache['unsorted_unique'] is sorted_unique
 
+class TestEmptyAtomGroup(object):
```

Comment:
```suggestion

class TestEmptyAtomGroup(object):
```
As above, slight PEP8 thing